.github/workflows/ci.yml

```diff
@@ -10,17 +10,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [ '7.2', '7.3', '7.4', '8.0' ]
+        php: [ '8.0', '8.1' ]
```

Comment:
I can handle that blocked (required) jobs, but I'm not quite sure why we're dropping PHP 7 support? 

I see 7.4 is used in more than 50%, see https://packagist.org/packages/php-translation/common/php-stats